### PR TITLE
PEP 373: Move 2.7.18rc1 to past

### PR DESCRIPTION
**Pull request by brainwane**
Toward https://github.com/python/steering-council/issues/3#issuecomment-613135126 .

Signed-off-by: Sumana Harihareswara <sh@changeset.nyc>

### CODE REVIEW

File: pep-0373.txt [hunk 1]
```diff
@@ -91,6 +90,7 @@ Dates of previous maintenance releases:
 - 2.7.16 2019-03-02
 - 2.7.17rc1 2019-10-05
 - 2.7.17 2019-10-19
+- 2.7.18 release candidate 2020-04-04
```
Comment (hugovk) [inline pep-0373.txt:93]: For consistency with others in this list:
```suggestion
- 2.7.18rc1 2020-04-04
```
Re: https://www.python.org/downloads/release/python-2718rc1/

File: pep-0373.txt [hunk 2]
```diff
@@ -15,11 +15,11 @@ Abstract
...
 Python 2.7.  The schedule primarily concerns itself with PEP-sized
-items.  Small features may be added up to and including the first beta
-release.  Bugs may be fixed until the final release.
+items.  Small features will no longer be added.  Bugs may be fixed
+until the final release.
 
-Update
-======
+Update (April 2014)
+===================
```
Comment (hugovk) [inline pep-0373.txt:22]: :+1: Good idea adding the date, it was a bit confusing earlier.

File: pep-0373.txt (hunk 1 above)
Comment (brainwane) [reply pep-0373.txt]: Thanks, good idea! Fixed.

File: pep-0373.txt (hunk 2 above)
Comment (brainwane) [reply pep-0373.txt:22]: Thanks!

**Comment by brainwane** [comment]
Thanks @hugovk .

Ready for re-review.

**Comment by brainwane** [comment]
cc @benjaminp

**Pull request merged**